### DISCUSSION TITLE
Cannot use Deepgram function

### DISCUSSION THREAD

**Discussion by luckylukezzz** [deepgram/community#371] [General help] [answered]
ImportError: cannot import name 'Deepgram' from partially initialized module 'deepgram' (most likely due to a circular import)

whats the issue causing this?
I tried unstalling  and reinstalling the sdk, still the same

**Comment by lukeocodes** [comment]
Hi @luckylukezzz, another Luke here 👋🏻 

Can you provide a little more detail? I assume this is using one of our SDKs. Can you let me know which one? Can you share the code you're using to import and initialise our SDK, and any usage of it?

Don't paste your API key 🙏🏻

**Reply by cowanAI** [reply]
same here, it only happens when I start a new project in Visual studio code, the SDK

**Reply by cowanAI** [reply]
but the original project works fine

**Reply by luckylukezzz** [reply]
Hello @cowanAI   check my answer below

**Comment by luckylukezzz** [comment, marked as answer]
Got it working , it was my ignorance :)
If someone met with this problem, I'll explain how I made it to work. (python api )
First you have to initialize it in a package ( make a folder, create an __init__.py file and add the deepgram key with the function Deepgram
example : inside deepgraminit (just a name) folder  
```python
#in __init__.py
from deepgram import Deepgram

DEEPGRAM_API_KEY=<your key here>
deepgram = Deepgram(DEEPGRAM_API_KEY)

async def transcribe_audio():

    audio=open("rec.wav", "rb")
    # Set the source
    source = {
      'buffer': audio,
      'mimetype': "wav"
    }
    try:
        response = await deepgram.transcription.prerecorded(source, {'punctuate': True})
        print(response['results']['channels'][0]['alternatives'][0]['transcript'])
    except Exception as e:
        print(f"An error occurred: {e}")
  ```

then , outside of the package,
```python
#at testing.py 
from deepgraminit import transcribe_audio
import asyncio
asyncio.run(transcribe_audio())
```
